marklogic-data-hub-central/ui/src/config/tooltips.config.tsx

```diff
@@ -0,0 +1,681 @@
+};
+
+const FlowMessages = {
+};
+
+
+/* --- STEPS - COMMON --- */
+
+const CommonStepTooltips = {
+  name: 'The name of this step definition.',
+  description: 'A description of this step definition.',
+  sourceQuery: 'The collection tag or CTS query that selects the source data to process in this step.',
+  sourceDatabase: 'The database where the input data is read from.',
+  targetDatabase: 'The database where to store the processed data.',
+  targetFormat: 'The format of the records in the target database.',
+  targetPermissions: (
+    <span>A comma-delimited string that defines permissions required to access the processed record. The string must be in the format 
+      <span style={{fontFamily: "monospace"}}> role, capability, role, capability,...</span>, where 
+      <i> capability</i> can be <span style={{fontFamily: "monospace"}}>read</span>, 
+      <span style={{fontFamily: "monospace"}}> insert</span>, 
+      <span style={{fontFamily: "monospace"}}> update</span>, or 
+      <span style={{fontFamily: "monospace"}}> execute</span>.</span>
+      ),
+  headers: 'A JSON object that represents additional metadata to add to the header section of the envelope of each record.',
+  additionalCollections: 'The collection tags to add to the default tags assigned to the processed record.',
+  provGranularity: 'The level of detail logged for provenance. Choose *coarse* for the default level or *off* for no provenance logging.',
+  validateEntity: 'Choose whether to validate each document against the entity type definition and how to handle documents with errors.',
+  batchSize : 'The maximum number of items to process in a batch.',
+  processors: 'Custom modules that perform additional processes after the core step processes are completed and before the results are saved.',
+  customHook: 'A custom module that performs additional processes in its own transaction before or after the core step transaction. Results are saved within a transaction.',
+  options: 'Key-value pairs to pass as parameters to the custom module.',
+  customModuleURI: 'The path to your custom step module.'
+};
+
+/* Note: Some of the following are intentionally duplicated in other *Placeholders constants. */
+const CommonStepPlaceholders = {
+  name: 'Enter the name.',
+  description: 'Enter the definition.',
+  sourceQuery: 'Enter a collection tag or CTS query.',
+  sourceDatabase: 'Choose where to get the data.',
+  targetDatabase: 'Choose where to store the processed data.',
+  targetFormat: 'Choose the format for the resulting documents.',
+  targetPermissions: 'Enter the permissions.',
+  headers: 'Enter additional header metadata.',
+  additionalCollections: 'Enter additional collection tags.',
+  provGranularity: 'Choose the level of provenance detail.',
+  validateEntity: 'Choose how to handle documents with errors.',
+  batchSize : 'Enter the batch size.',
+  processors: 'Enter the array of processor settings.',
+    /* Can we use an example like this?
```

Comment:
Let's not keep the code that is commented out and we don't need for now. Looks like this is a question for PM & UX team to include a sample processor and Custom hook in the tooltip and once its approved then we can add it later.